torch/fx/graph_module.py

```diff
@@ -194,10 +194,19 @@ def __reduce__(self):
     def __deepcopy__(self, memo):
         fake_mod = torch.nn.Module()
         fake_mod.__dict__ = copy.deepcopy(self.__dict__)
-        return GraphModule(fake_mod, self.graph)
+        # delete generated attributes before creating a new
+        # GraphModule
+        generated_attrs = ['code', '_graph', '_modules']
+        for attr in generated_attrs:
```

Comment:
`generated_attrs` is only used once, so you can probably just put the list literal in the loop statement